molgenis/molgenis-emx2
# fix: design is hidden when values are provided

how to test:
- see that design block is shown when e.g. 'type' or 'design' has been filled it but no mainmedicalcondition
